src/react/src/util/styles.js

```diff
@@ -654,6 +654,78 @@ export const makeDownloadExcelButtonStyles = theme =>
         }),
     });
 
+export const makeDashboardContributionRecordStyles = theme =>
+    Object.freeze({
+        loaderStyles: Object.freeze({
+            display: 'block',
+            margin: 'auto',
+        }),
+        emptyBlockStyles: Object.freeze({
+            height: '100px',
+            display: 'flex',
+            justifyContent: 'center',
+            alignItems: 'center',
+        }),
+        emptyTextStyle: Object.freeze({ color: 'rgb(142 138 138 / 87%)' }),
+        buttonStyles: Object.freeze({
+            margin: '20px',
+            padding: '20px',
+            width: '100%',
+            fontWeight: 'bold',
+        }),
+        container: {
+            marginBottom: '25px',
+            width: '100%',
+            padding: '20px',
+        },
+        title: {
+            paddingBottom: '20px',
+        },
+        listItemStyle: {
+            display: 'flex',
+            flexDirection: 'row',
+            justifyContent: 'space-between',
+        },
+        listItemTextStyle: {
+            padding: 0,
+        },
+        dividerStyle: {
+            height: '3px',
+            backgroundColor: 'rgb(0 0 0 / 55%)',
```

Comment:
Please add this color to the COLOURS.js.

Reply:
replaced with COLOURS.DARK_GREY